src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -119,15 +124,32 @@ private File getTmpWrapperScript(SchemaType schemaType, File workingDir) {
     }
 
     public void createDbScriptTable() {
-        File scriptsDirectory = appArguments.getScriptsDirectory();
-        String tmpFileName = new Date().getTime() + CREATE_SQL;
+        int exitCode = executeResourceScript(CREATE_SQL, false);
+        if (exitCode != EXIT_CODE_SUCCESS) {
+            throw new ScriptExecException("Can't create DB objects used by SCMDB.");
+        }
+    }
+
+    public void executeCompileSchemas() {
+        int exitCode = executeResourceScript(COMPILE_SCHEMAS_SQL, true);
+        if (exitCode != EXIT_CODE_SUCCESS) {
+            throw new ScriptExecException("Can't compile invalid objects in _user, _rpt, _pkg schemas.");
+        }
+    }
+
+    private int executeResourceScript(String scriptFileName, boolean isCompileSchema) {
+        return executeResourceScript(appArguments.getScriptsDirectory(), scriptFileName, isCompileSchema);
+    }
+
+    private int executeResourceScript(File scriptsDirectory, String scriptFileName, boolean isCompileSchema) {
```

Comment:
refactor method signature to `executeResourceScript(String scriptFileName, String  errorMessage) {`, so call of this method should looks like 
<img width="1228" alt="image" src="https://user-images.githubusercontent.com/3282060/210522288-e3b49294-5fd7-424f-b40a-8851fb6b15ef.png">

Reply:
the method is refactored but a parameter with name '**isCompileSchema**' is not removed because the parameter is needed inside the method in a row "int exitCode = execute(sqlScript, **isCompileSchema**);"